src/gui/owncloudsetupwizard.cpp

```diff
@@ -353,7 +353,35 @@ void OwncloudSetupWizard::testOwnCloudConnect()
     // so don't automatically follow redirects.
     job->setFollowRedirects(false);
     job->setProperties(QList<QByteArray>() << "getlastmodified");
-    connect(job, &PropfindJob::result, _ocWizard, &OwncloudWizard::successfulStep);
+    connect(job, &PropfindJob::result, this, [this, account] {
```

Comment:
I would prefer if the code for triggering and handling the request checking the server here was a separate function. At least it must have a comment explaining what's going on.